app/controllers/search_terms_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SearchTermsController < ApplicationController
+  def index
+  end
+  
+  # GET /search_terms
+  def index
+    @search_terms = SearchTerm.all
+  end
+
+  def import
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/1d081ecd36c48dd47f0123e7164aa066b7a4d685/app/controllers/search_terms_controller.rb#L10"></span>Assignment Branch Condition size for import is too high. [&lt;0, 19, 3&gt; 19.24/17]</td>
    </tr>
  </tbody>
</table>